src/lcec_conf.h

```diff
@@ -29,7 +29,7 @@
 #define LCEC_CONF_STR_MAXLEN 48
 
 #define LCEC_CONF_SDO_COMPLETE_SUBIDX -1
-#define LCEC_CONF_GENERIC_MAX_SUBPINS 32
+#define LCEC_CONF_GENERIC_MAX_SUBPINS 48
```

Comment:
Is this in corelation to the 4104 stuff or should we better open a new request to increase the constant?